Applications/SlicerApp/Testing/Python/DICOMReaders.py

```diff
@@ -97,23 +97,27 @@ def test_Part1DICOM(self):
         "fileName": "Mouse-MR-example-where-GDCM_fails.zip",
         "name": "Mouse-MR-example-where-GDCM_fails",
         "seriesUID": "1.3.6.1.4.1.9590.100.1.2.366426457713813178933224342280246227461",
-        "expectedFailures": ["GDCM", "Archetype"]
+        "expectedFailures": ["GDCM", "Archetype"],
+        "voxelValueQuantity": "(DCM, 110852, MR signal intensity)",
+        "voxelValueUnit": "(UCUM, [hnsf'U], Hounsfield unit)"
```

Comment:
What sounds right to me is that when there's only one thing being measured then you use a 'unit of measure' or a 'unit' for short.  It would sound right use 'units of measure' when there's more than one unit involved, like the units of measure in the metric system.  It's a little fuzzy when you have a compound unit, like meters per second, where there are multiple units involved.  Or when you have a plural quantity, like there are "three units of blood".  

Anyway, I think we should follow DICOM Data Dictionary conventions which uses Units in every case, even when there's only one involved.  For example DeviceDiameterUnits clearly has only one unit of measure specified (VM=1) and there are many similar cases and only a couple exceptions.